docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -8,6 +8,26 @@ ifdef::notBuildingForSite[]
 include::{includedir}/menu.adoc[]
 endif::notBuildingForSite[]
 
+== Getting Ehcache
+
+From Maven
+
+    <dependencies>
+        <dependency>
+          <groupId>org.ehcache</groupId>
+          <artifactId>ehcache</artifactId>
+          <version>3.10.0</version>
+        </dependency>
+    </dependencies>
+
+From Gradle
+
+    org.ehcache:ehcache:3.10.0
```

Comment:
Although this is a valid short form syntax for Gradle, it's probably better if we use the more complete:
```asciidoc
[source,groovy]
----
dependencies {
  implementation 'org.ehcache:ehcache:3.10.0'
}
----
```

Reply:
Accommodated this review comment and push the changes